router/admin.go

```diff
@@ -127,7 +128,7 @@ func (r *RouterRpcHandler) GetDSStats(dsName string, result *string) (err error)
 	if err != nil {
 		return err
 	}
-	defer dbHandle.Close()
+	defer func() { _ = dbHandle.Close() }()
```

Comment:
What was the problem with the previous approach?

Reply:
Ignoring errors bothers linters, in this case we could issue a warning or simply ignore. By making the assignment explicit we're making clear that we're OK with ignoring a DB connection closure error.

Reply:
If we are doing this, then why not put error log also. All the places?

Reply:
> If we are doing this, then why not put error log also. All the places?

One bit at a time I'm sure we'll make it consistent :+1: 

In regards to the error log I'd say that usually nobody really does anything when they can't close a connection so it would just mean polluting the logs. If we try to close it more than once for example we could get `sql: duplicate driverConn close` but it's not a big deal. If you feel like having those in the logs can benefit someone then perhaps I can raise a `Warn` instead of an `Error`. Thoughts?

Reply:
Agree with log pollution.